shell/src/main/java/alluxio/cli/ValidateEnv.java

```diff
@@ -87,80 +93,108 @@
   private static final List<ValidationTask> MASTER_TASKS = new ArrayList<>();
   private static final List<ValidationTask> WORKER_TASKS = new ArrayList<>();
 
-  private static final AlluxioConfiguration CONF;
+  private final AlluxioConfiguration mConf;
+  private final String mPath;
 
-  static {
-    CONF = InstancedConfiguration.defaults();
```

Comment:
Use a constructor now instead of static block, as the validation can be against a nested mount.